packages/core-app-api/src/app/types.ts

```diff
@@ -213,6 +214,20 @@ export type AppOptions = {
     }
   >;
 
+  /**
+   * A map of plugin package names to internal groups (as entity refs defaulting
+   * to kind Group).
+   *
+   * The package name (key) can contain wildcards and will be matched using
+   * minimatch.
+   */
+  pluginOwners?: Record<string, string>[];
+
+  /**
+   * Decorate the PluginInfo part of a plugin
+   */
+  pluginInfoDecorator?: (plugin: PluginInfo, id: string) => void;
```

Comment:
I'm not entirely sure about the split of these tbh. Could it perhaps be achieved with a single option that can be applied in different ways? Or perhaps at least group them up a bit, thinking something like:

```ts
createApp({
  pluginInfo: {
    owners: ...,
    decorator: ...,
  }
})
```

Reply:
Certainly. Not if the two older should be `_experimental*` prefixed though, let me push a new version of this, as you mention here - it makes sense.

Reply:
Not seeing any change here? 😅

Reply:
Fixed